modules/installation-guide/partials/proc_configuring-labels-for-routes-ingresses.adoc

```diff
@@ -0,0 +1,40 @@
+
+[id="configuring-labels-for-routes_ingresses_{context}"]
+= Configuring labels for routes or ingresses
+
+This procedure describes how to configure labels for routes or ingresses to organize and categorize (scope and select) objects.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* {prod-short} is deployed in a Kubernetes cluster.
+
+IMPORTANT: It is needed to use comma to separate labels: `key1=value1,key2=value2`
+
+.Procedure
+
+. To configure labels for routes update the Custom Resource with the following commands:
```

Comment:
As far as I understood, Kubernetes <=> Ingresses and OpenShift <=> Routes. Is that right?

Then we should have 2 procedures to distinguish the 2 distinct scenarios:

* Configuring labels for OpenShift Routes using the CLI
* Configuring labels for Kubernetes Ingresses using the CLI

Additionally, it would be awesome to have one procedure describing the usage of the WebUI, probably like:

* Configuring labels for OpenShift Routes using the OpenShift Web Console

Reply:
Divided into 2 procedures.

Reply:
The procedure describing the usage of the WebUI make sense only for OpenShift and it will be much longer then using CLI.
I don't think it worth describing it.